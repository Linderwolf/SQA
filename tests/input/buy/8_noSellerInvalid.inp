login
FullStandard
buy
Game1
AB