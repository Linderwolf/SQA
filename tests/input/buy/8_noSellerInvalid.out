Welcome! 
Please enter your username:
Username accepted! How may we help you today? :
Please enter the name of the game you would like to buy:
Reading the Available Games File...
Please enter the name of the seller of Game1:
Reading the Available Games File...
No such seller exists. Please enter the correct seller's name: